config/cable.yml

```diff
@@ -0,0 +1,9 @@
+development:
```

Comment:
What do you think about removing this file? If we want to add ActionCable, we can add this file then